lib/lita/handlers/lunch_reminder.rb

```diff
@@ -517,10 +518,9 @@ def get_winners_msg_timestamp
         redis.get('winners_msg_timestamp')
       end
 
-      def get_lunch_buttons(message)
+      def get_lunch_buttons()
```

Comment:
Method has too many lines. [17/15]<br>Omit the parentheses in defs when the method doesn't accept any arguments.